torch/csrc/autograd/functions/convolution.cpp

```diff
@@ -327,6 +365,126 @@ auto ConvBackward::releaseVariables() -> void {
   bias_.data.reset();
 }
 
+
+// ConvBackwardBackward implementation
+
+auto ConvBackwardBackward::apply(const variable_list& grad_grad_inputs) -> variable_list {
+  if (grad_grad_inputs.size() != 3) throw std::runtime_error("expected three grad_grad_inputs");
+  if (transposed) throw std::runtime_error("ConvBackwardBackward does not support transposed convolution");
+
+  auto ggI = grad_grad_inputs[0];
+  auto ggW = grad_grad_inputs[1];
+  auto ggb = grad_grad_inputs[2];
+
+  auto gO = grad_output_.unpack();
+  auto weight = weight_.unpack();
+
+  // Compute ggO = conv(w, ggI) + conv(ggW, i) + ggb
+  std::shared_ptr<Variable> ggO = nullptr;
+  if (ggI) {
+    ggO = std::make_shared<ConvForward>(*this)->apply({ggI, weight, nullptr})[0];
+  }
+
+  if (ggW) {
+    auto ggW_term = std::make_shared<ConvForward>(*this)->apply({input_.unpack(), ggW, nullptr})[0];
+    if (ggO) {
+      ggO = std::make_shared<Add>()->apply({ggO, ggW_term})[0];
+    } else {
+      ggO = ggW_term;
+    }
+  }
+
+  if (ggb) {
+    // View as (1, ggb.size(0), 1, 1...)
+    std::vector<long> new_size(gO->data->sizes().size(), 1);
+    new_size[1] = ggb->data->sizes()[0];
+    auto ggb_contiguous = std::make_shared<Clone>()->apply({ggb})[0];
```

Comment:
The Contiguous function does not exist right now, i need to create it. But what should the backward of Contiguous be? Identity or Contiguous?